crossterm_input/examples/async_input.rs

```diff
@@ -1,9 +1,16 @@
 extern crate crossterm_input;
+// this is included to enable raw mode
+extern crate crossterm_screen;
+use crossterm_screen::Screen;
+// this is included to enable writing in raw mode
+extern crate crossterm_utils;
+use crossterm_utils::write;
+use std::fmt::Write;
```

Comment:
Can you move this to line 12? 

I like to have std imports and crate imports grouped makes it easier to read.

```
use crossterm::input::{*.....}

use std::fmt::Write
use std::io::Read;
use std::{thread, time};
```